11-spatial-cv.Rmd

```diff
@@ -56,7 +56,7 @@ Building on this the chapter implements a more typical machine learning algorith
 The models' **predictive performance** will be assessed using spatial cross-validation (CV), which accounts for the fact that geographic data is special.
 
 CV determines a model's ability to generalize to new data, by splitting a dataset (repeatedly) into training and test sets.
-It uses the training data to fit the model, and checks its performance when predicting to the test data.
+It uses the training data to fit the model, and checks its performance when predicting against the test data.
```

Comment:
@jannes-m you happy with this? Will accept this PR and fix the 2 mini issues I commented on below. Many thanks Katy!